ci/serverless/common.sh

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env bash
+set -ex
+
+export CURRENT_DIR="$(dirname "$0")"
+export INDEX_NAME="serverless_it_${BUILDKITE_BUILD_NUMBER:-`date +%s`}"
+# store all error messages
+export ERR_MSGS=()
+# numeric values representing the results of the checks. 0: pass, >0: fail
+export CHECKS=()
+
+setup_vault() {
+  vault_path=secret/ci/elastic-logstash/serverless-test
+  set +x
+  export ES_ENDPOINT=$(vault read -field=es_host "${vault_path}")
+  export ES_USER=$(vault read -field=es_user "${vault_path}")
+  export ES_PW=$(vault read -field=es_user_pw "${vault_path}")
+  export KB_ENDPOINT=$(vault read -field=kb_host "${vault_path}")
+  set -x
+}
+
+build_logstash() {
+  ./gradlew clean bootstrap assemble installDefaultGems
+}
+
+prepare_test_data() {
```

Comment:
let's use a verb that better indicates the kind of preparation we're doing (indexing data to ES):

```suggestion
index_test_data() {
```